tests/Copy-DbaTableData.Tests.ps1

```diff
@@ -5,15 +5,18 @@ Write-Host -Object "Running $PSCommandpath" -ForegroundColor Cyan
 Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
     BeforeAll {
         $db = Get-DbaDatabase -SqlInstance $script:instance1 -Database tempdb
+        $db2 = Get-DbaDatabase -SqlInstance $script:instance2 -Database tempdb
```

Comment:
@road42 we are not allowed to do this with Appveyor. We cannot connect between two VMs in that environment.